extension.js

```diff
@@ -151,49 +157,55 @@ var ContainerSubMenuMenuItem = GObject.registerClass(
-                stopMenuItem.insert_child_at_index(createIcon("process-stop-symbolic", "status-stopped"), 1);
-                this.menu.addMenuItem(stopMenuItem);
-                const restartMenuItem = new ContainerMenuItem(container.name, "restart", () => container.restart());
-                restartMenuItem.insert_child_at_index(createIcon("system-reboot-symbolic", "status-restart"), 1);
-                this.menu.addMenuItem(restartMenuItem);
-                this.menu.addMenuItem(createTopMenuItem(container));
-                this.menu.addMenuItem(createShellMenuItem(container));
-                this.menu.addMenuItem(createStatsMenuItem(container));
                 break;
             }
             case "Paused":
             case "paused": {
                 this.insert_child_at_index(createIcon("media-playback-pause-symbolic", "status-paused"), 1);
-                const unpauseMenuItem = new ContainerMenuItem(container.name, "unpause", () => container.unpause());
-                unpauseMenuItem.insert_child_at_index(createIcon("media-playback-start-symbolic", "status-start"), 1);
-                this.menu.addMenuItem(unpauseMenuItem);
                 break;
             }
             default:
                 this.insert_child_at_index(createIcon("action-unavailable-symbolic", "status-undefined"), 1);
                 break;
             }
 
-            // add log button
-            const logMenuItem = createLogMenuItem(container);
-            this.menu.addMenuItem(logMenuItem);
+            details.actor.add_child(new ContainerMenuItem(container.name, "Logs", () => container.logs()));
+            details.actor.add_child(new ContainerMenuItem(container.name, "Top", () => container.watchTop()));
+            details.actor.add_child(new ContainerMenuItem(container.name, "Shell", () => container.shell()));
+            details.actor.add_child(new ContainerMenuItem(container.name, "Statistics", () => container.stats()));
+
+            details.actor.add_child(new PopupMenu.PopupSeparatorMenuItem());
+            details.actor.add_child(new PopupMenuItem("Status", container.status));
+            if (container.startedAt !== null) {
+                details.actor.add_child(new PopupMenuItem("Started", container.startedAt));
+            }
+            details.actor.add_child(new PopupMenuItem("Image", container.image));
+            details.actor.add_child(new PopupMenuItem("Command", container.command));
+            details.actor.add_child(new PopupMenuItem("Created", container.createdAt));
+            details.actor.add_child(new PopupMenuItem("Ports", container.ports));
+            const ipAddrMenuItem = new PopupMenuItem("IP Address", "");
+            details.actor.add_child(ipAddrMenuItem);
+            this.inspected = false;
+
+            // add more stats and info - inspect - SLOW
+            this.connect("button_press_event", () => {
```

Comment:
```suggestion
            this.connect("button-press-event", () => {
```

Underscores work, but [are discouraged](https://gitlab.gnome.org/GNOME/glib/-/blob/main/gobject/gsignal.c#L439)

Reply:
ack, changed in 3 places